projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -855,6 +883,52 @@ function wpcom_launchpad_get_newsletter_subscriber_count() {
 	return (int) $total_subscribers;
 }
 
+/**
+ * Determines if Stripe has been connected.
+ *
+ * @return bool Whether Stripe account is connected.
+ */
+function wpcom_launchpad_is_stripe_connected() {
+	$membership_settings = wpcom_launchpad_get_membership_settings();
+	if ( ! $membership_settings ) {
+		return false;
+	}
+	return isset( $membership_settings['connected_account_id'] ) && $membership_settings['connected_account_id'] !== '';
+}
+
+/**
+ * Determines if any paid membership plan exists.
+ *
+ * @return bool Whether paid plan exists.
+ */
+function wpcom_launchpad_has_paid_membership_plans() {
+	$membership_settings = wpcom_launchpad_get_membership_settings();
+	if ( ! $membership_settings ) {
+		return false;
+	}
+	return isset( $membership_settings['products'] ) && is_array( $membership_settings['products'] ) && ( count( $membership_settings['products'] ) > 0 );
+}
+
+/**
+ * Get membership settings.
+ *
+ * @return array|null Membership settings or null.
+ */
+function wpcom_launchpad_get_membership_settings() {
+	$is_atomic = defined( 'IS_ATOMIC' ) && IS_ATOMIC;
+
+	// Memberships lib is only available on Simple sites.
+	// A follow up will fetch membership settings for Atomic.
```

Comment:
Minor note: the code _and_ the data are only present and directly accessible from WPCOM Simple.

It looks like we should be able to use code very similar to the code block below to fetch this data when we're in an Atomic context. That code is for the `wpcom/v2/memberships/status` endpoint, which will make a proxied API call to WordPress.com when we're not running on WPCOM Simple.
https://github.com/Automattic/jetpack/blob/cce10b2a0c92b53b79bda816e8fcd3899fdb30ad/projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/memberships.php#L369-L401